store/extensions/GPT Reply/extension.json

```
@@ -0,0 +1,38 @@
+{
+    "title": "GPT Reply",
+    "description": "GPT Reply lets you reply to other people with Claude or ChatGPT in Origins.",
+    "authors": [
+      {
+        "name": "QDave",
+        "discord": "qdave"
+      }
+    ],
+    "version": "1.0",
+    "categories": ["Usability"],

```

It doesn't really fit the description of the Usability category, the Fun category is more fitting
```suggestion
    "categories": ["Fun"],
```